### PR TITLE
Integration features

### PR DESCRIPTION
**Pull request by cdsandoval**

### CODE REVIEW

**Review by valevassallo — APPROVED** [review]

File: resources/js/index.js [hunk 1]
```diff
@@ -47,3 +46,29 @@ function toggle_complete(element) {
...
   sessionStorage.setItem("list", list_task);
 }
+
+function todo_list() {
+  let element = save_task();
+  let string_to_date = moment(element.due_date).format();
+  const formatted_due_date = moment(string_to_date).format("L");
+  const formatted_creation_date = moment(element.creation_date).format("L");
+  const listContainer = document.getElementById("list1");
+
```
Comment (liannivin) [inline resources/js/index.js]: Could you explain what means this type of format, please?

File: index.html [hunk 2]
```diff
@@ -73,6 +73,7 @@
     <!-- FOOTER -->
     <div class="footer">Created by Team X</div>
     <!-- END FOOTER -->
+    <script src="https://rawgit.com/moment/moment/2.2.1/min/moment.min.js"></script>
```
Comment (dotconde) [inline index.html:76]: Cool JS clocker library

File: resources/js/index.js [hunk 3]
```diff
@@ -47,3 +46,29 @@ function toggle_complete(element) {
...
+  listContainer.insertAdjacentHTML(
```
Comment (dotconde) [inline resources/js/index.js:57]: First time I see an adjancentHTML 😮

File: resources/js/index.js (hunk 1 above)
Comment (valevassallo) [reply resources/js/index.js:56]: do you mean the L format? If that's the case, then, according to the Moment library, L means showing the date like this: "05/20/2019"

File: index.html (hunk 2 above)
Comment (valevassallo) [reply index.html:76]: thank you :smile: :ok_hand:

File: resources/js/index.js (hunk 3 above)
Comment (Linzeur) [reply resources/js/index.js:57]: That method works for add new content to element in DOM, I remark "ADD"

**Pull request merged**